dat/events/neutral/news.lua

```diff
@@ -1,238 +1,115 @@
 --[[
--- Event for creating generic news
+-- Event for creating news
 --
 --]]
 
 require "numstring.lua"
 require "jumpdist.lua"
 
 
-header_table={}
```

Comment:
Why were the headers removed?

Reply:
We removed those because it seemed we couldn't guarantee them to show up on top, or at least based on how it was put together before that's how it looked. We'll do some experimenting with that.

🕵️

Reply:
Figured it out, so it's the date, huh? Added the headers back in, this time in a more robust fashion (no more duplicates... at least for new saves, I'll mention more on that in the main conversation).

🕵️